client/my-sites/marketing/main.jsx

```diff
@@ -123,6 +126,22 @@ export const Sharing = ( {
 		titleHeader = translate( 'Integrations' );
 	}
 
+	// Determine the right type of upsell according to the plan.
+	// For the Free plan and the Starter plan, we want to upsell the no-ads add-on.
+	// Otherwise, it upsells the Pro plan.
+	let upsellNudgeTitle = translate( 'No ads with WordPress.com Pro' );
+	let upsellNudgeDesc = translate( 'Prevent ads from showing on your site.' );
+	let upsellNudgeHref;
+
+	if (
+		isStarterPlanEnabled() &&
+		( isFreePlan( currentPlanSlug ) || isStarterPlan( currentPlanSlug ) )
+	) {
+		upsellNudgeTitle = translate( 'Remove ads from your site with the No-Ads add-on' );
```

Comment:
The name of the product is actually "No Ads", not "No-Ads".

Reply:
Changed.
Thanks for catching this.